packages/core/src/internal/data-grid/data-grid-lib.ts

```diff
@@ -155,6 +155,18 @@ export function getStickyWidth(
     return result;
 }
 
+export function getFreezeTrailingHeight(
+    rows: number,
+    freezeTrailingRows: number,
+    getRowHeight: number | ((row: number) => number)
+): number {
+    let result = 0;
+    for (let i = rows - freezeTrailingRows; i < rows; i++) {
+        result += typeof getRowHeight === "number" ? getRowHeight : getRowHeight(i);
```

Comment:
This is inefficient for large numbers of freeze regions, simple multiplication can work anytime the rowHeight is fixed which is the vast majority of cases.